File: Prod.RutaDigital.Entidades/Evento/EventoResponse.cs
namespace Prod.RutaDigital.Entidades
{
    public class EventoResponse
    {
        public int id_evento { get; set; }
        public int id_tipo_modalidad { get; set; }
        public string? foto { get; set; }
        public string? nombre { get; set; }
        public string? descripcion_corta { get; set; }
        public string? descripcion { get; set; }
        public string? lugar { get; set; }
        public string? direccion { get; set; }
        public string? url_evento { get; set; }
        public DateTime? fecha_evento { get; set; }
        public string? modalidad { get; set; }
        public double latitud { get; set; }
        public double longitud { get; set; }
        public string? plataforma { get; set; }
        public byte[]? numArray { get; set; }
    }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/eventos/interfaces/evento.response.ts
import { SafeUrl } from '@angular/platform-browser';

export interface EventoResponse {
  // foto: string;
  id_evento?: number;
  id_tipo_modalidad?: number;
  foto?: SafeUrl;
  nombre?: string;
  descripcion_corta?: string;
  descripcion?: string;
  lugar?: string;
  direccion?: string;
  url_evento?: string;
  fecha_evento?: Date;
  modalidad?: string;
  latitud?: number;
  longitud?: number;
  plataforma?: string;
  numArray?: Blob;
  imagenEvento?: SafeUrl;
}
